app/src/main/java/com/pavesid/androidacademy/ui/activities/MovieDetailsActivity.kt

```diff
@@ -0,0 +1,62 @@
+package com.pavesid.androidacademy.ui.activities
+
+import android.graphics.drawable.Drawable
+import android.os.Bundle
+import android.widget.ImageView
+import androidx.appcompat.app.AppCompatActivity
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import coil.load
+import com.pavesid.androidacademy.R
+import com.pavesid.androidacademy.model.Actor
+import com.pavesid.androidacademy.ui.adapters.CastAdapter
+import com.pavesid.androidacademy.ui.decorations.MarginItemDecoration
+
+class MovieDetailsActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_movie_details)
+
+        val orig = findViewById<ImageView>(R.id.orig)
+        orig.load(Drawable.createFromStream(assets.open("orig.png"), null))
+        val recycler = findViewById<RecyclerView>(R.id.cast_recycler)
```

Comment:
Actually this task does not mean to use RecyclerView but I agree that it is right decision here.